specutils/manipulation/utils.py

```diff
@@ -152,3 +152,42 @@ def excise_region(spectrum, region, exciser=linear_exciser):
     #
 
     return exciser(spectrum, region)
+
+
+def spectrum_from_model(model_input, spectrum):
+    """
+    This method will create a `~specutils.Spectrum1D` object
+    with the flux defined by calling the input ``model``. All
+    other parameters for the output `~specutils.Spectrum1D` object
+    will be the same as the input `~specutils.Spectrum1D` object.
+
+    Parameters
+    ----------
+    model : `~astropy.modeling.Model`
+        The input model or compound model from which flux is calculated.
+
+    spectrum : `~specutils.Spectrum1D`
+        The `~specutils.Spectrum1D` object to which the smoothing will be applied.
```

Comment:
Copy-and-paste error?  Probably should be "The `~specutils.Spectrum1D` object to use as the model template" or something like that.

Reply:
👍